templates/firefox-ios.xliff

```diff
@@ -1319,6 +1344,102 @@
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.SavePageAs" xml:space="preserve">
+        <source>Save Page As...</source>
+        <note>A label indicating the keyboard shortcut of saving the current web page in a format of the user's choice. This label is displayed in the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.Section.Bookmark" xml:space="preserve">
+        <source>Bookmarks</source>
+        <note>A label indicating a grouping of related keyboard shortcuts describing actions a user can do with Bookmarks. This label is displayed inside the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.Section.Edit" xml:space="preserve">
+        <source>Edit</source>
+        <note>A label indicating a grouping of related keyboard shortcuts describing actions a user can do within a web page. This label is displayed inside the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.Section.File" xml:space="preserve">
+        <source>File</source>
+        <note>A label indicating a grouping of related keyboard shortcuts describing actions a user can take on, and within, a Tab. This label is displayed inside the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.Section.History" xml:space="preserve">
+        <source>History</source>
+        <note>A label indicating a grouping of related keyboard shortcuts describing actions a user can do with History. This label is displayed inside the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.Section.Tools" xml:space="preserve">
+        <source>Tools</source>
+        <note>A label indicating a grouping of related keyboard shortcuts describing actions a user can do with locally saved items. This label is displayed inside the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.Section.View" xml:space="preserve">
+        <source>View</source>
+        <note>A label indicating a grouping of related keyboard shortcuts describing actions a user can do regarding the viewing experience of a webpage. This label is displayed inside the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.Section.Window" xml:space="preserve">
+        <source>Window</source>
+        <note>A label indicating a grouping of related keyboard shortcuts describing actions a user can take when navigating between their availale set of tabs. This label is displayed inside the Discoverability overlay when a user presses the Command key.</note>
```

Comment:
Same question as above
Perhaps these comments I flag need to be tweaked a bit so localizers understand the context better, and can better localize